git/test/test_index.py

```diff
@@ -838,6 +838,19 @@ def test_add_a_file_with_wildcard_chars(self, rw_dir):
         r.index.add([fp])
         r.index.commit('Added [.exe')
 
+    def test__to_relative_path_at_root(self):
+        root = osp.abspath(os.sep)
+
+        class Mocked(object):
+            bare = False
+            git_dir = root
+            working_tree_dir = root
+
+        repo = Mocked()
+        path = os.path.join(root, 'file')
+        index = IndexFile(repo)
+        index._to_relative_path(path)
```

Comment:
Is there an assertion missing? It looks like `_to_relative_path` returns the relative path, which is not verified here.

Reply:
Ah, right. Thank you, fixed in a new version.